uwblueprint/sistema
# David/file upload

# Notion ticket link

[Implement File Uploading](https://www.notion.so/uwblueprintexecs/Implement-CRUD-for-Google-Drive-API-69b22dcc28f24256be626080e85e67f9?pvs=4)



## Implementation description
- Created simple UI for file uploading
- Implemented NEXTJS routes for uploading to google drives
- Created Google Project to facilitate Create operations



## Steps to test

1. Pull new env variables from vault
2. Navigate to localhost:3000/page and upload a file
3.  Check sistema@blueprint.org to ensure that the file is in the lesson plan folder



## What should reviewers focus on?
- Cleanliness of code
- Ability for team members to understand and build on top of existing code

## Checklist
